_plugins/docs_generator.rb

```diff
@@ -168,7 +171,7 @@ def build_with_sphinx(repo_name, repo_path, repo_data)
     end
 
     Dir.glob(File.join(output_path, "{_images/*.*,_downloads/**/*.*}"),
-             File::FNM_CASEFOLD).each do |static_file_path|
+             File::FNM_PATHNAME).each do |static_file_path|
```

Comment:
Was the casefold flag removed on purpose? I would have expected
`File::FNM_CASEFOLD | File::FNM_PATHNAME`.

Reply:
So, `File::FMN_CASEFOLD` was (and is) a bit unnecessary as I don't expect `File.join(output_path, "{_images,_downloads}")` to change case. It doesn't do any harm either though.

Reply:
If it's not needed best not to confuse the issue :+1: